serverless/jobs/how-to/run-a-job.mdx

```diff
@@ -0,0 +1,41 @@
+---
+meta:
+  title: How to run a Serverless Job
+  description: This page explains how to run a Serverless Job
+content:
+  h1: How to run a Serverless Job
+  paragraph: This page explains how to run a Serverless Job
+  tags: run execute start serverless job scaleway
+dates:
+  validation: 2023-11-14
+  posted: 2021-11-14
+categories:
+  - serverless
+  - jobs
+---
+
+This page shows you how to manually run a Serverless Job from the Scaleway console.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a job](/serverless/jobs/how-to/create-job-from-scaleway-registry/)
+</Message>
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The Serverless Jobs page displays.
+
+2. Click the name of the job you want to run. The job **Overview** tab displays.
+
+3. Click the **Run job** button. A new entry appears in the **Job runs** section.
+
+The **Job runs** section contains basic monitoring information for your jobs:
+
+- A status, that can be `queued`, `ongoing`, `canceled`, or `succeeded`
+- A unique identifier (UID)
```

Comment:
```suggestion
- An unique identifier (UID)
```

Reply:
Weirdly it is actually `A unique identifier`

You would say "a unique identifier", "a uniform", "a unicorn", but "an umbrella" or "an uncle". No idea what the rule is here, perhaps a non-native speaker knows better than me :sweat_smile: